templates/publisher/market.html

```diff
@@ -34,8 +34,8 @@ <h1 class="u-no-margin--top">{{ title }}</h1>
 
           <div class="row">
             <div class="col-12 u-align--right">
-                <input type="submit" class="p-button--neutral" name="submit_revert" value="Revert"/>
-                <input type="submit" class="p-button--positive" name="submit_apply" value="Apply"/>
+                <input type="submit" class="p-button--neutral js-market-revert" name="submit_revert" value="Revert"/>
```

Comment:
If you change this type to `reset` it should reset the form and `return` should submit the form as expected.

Reply:
It seems that Vanilla doesn't like reset buttons (`display: none`).
Also, it will only reset text fields and not icon/screenshots that are custom.

This is not a new issue so if possible I would merge it as it is and think of a better solution in next PR.
Possibly I guess we can make it just a button with click handler that would reload the page (without posting).

Reply:
:+1: